### PR TITLE
Update readme for paper-scroll-header-panel

### PR DESCRIPTION
**Pull request by blasten**
- Add content to the readme
- Change selector for the title of `paper-toolbar`

cc @cdata @morethanreal  @notwaldorf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
...
+```
+
+To change the background for toolbar when it is condensed:
+```css
+paper-scroll-header-panel {
+  --paper-scroll-header-panel-condensed-header: {
+    background-color: #f4b400;
+  };
+}
+```
```
Comment (cdata) [inline README.md]: Newline?

**Comment by cdata** [comment]
Just curious: why was the attribute deemed better than the class? It seems like they are used the same way in most cases.

**Comment by blasten** [comment]
@cdata  I was chatting with @frankiefu about using attribute selectors to identify subparts of an element. For example,  `paper-drawer-panel` has a `[drawer]` and `[main]`.  We did something similar for `paper-toolbar`, but we aren't done yet. We still need to do some work around `class="bottom"` for specifying content location.  I would like to sit down with you  guys to discuss this API:

``` html
<paper-toolbar>
  <div title>Title</div>
</paper-toolbar>
```

``` html
<paper-toolbar>
  <div bottom>
    <div title>Title</div>
  </div>
</paper-toolbar>
```

You could still use classes for styling and layout and attributes for subparts of the element. A subpart may or may not have a default style.

**Comment by cdata** [comment]
Ergonomically speaking, I like the idea of using the attributes to identify sub-parts of an element. However, it is technically not valid HTML to do this for non-custom-elements. Are we doing it for any reasons other than the fact that it "feels" good?

**Comment by cdata** [comment]
Also, if you really don't want to use a class, would the `id` attribute be a sufficient alternative? If not, why not?

**Comment by blasten** [comment]
@cdata  could you check it again?  I'm just adding the docs

File: README.md (hunk 1 above)
Comment (cdata) [inline README.md]: Still missing a newline here.

**Comment by cdata** [comment]
Please fix the newline per feedback, otherwise LGTM :+1:

**Comment by blasten** [comment]
:+1:

**Pull request merged**